lib/pipeline.js

```diff
@@ -424,45 +424,58 @@ class PipelineModel extends BaseModel {
                     ref: prInfo.ref,
                     isPR: true
                 }),
-                this.jobs
+                this.jobs,
+                Promise.resolve(prInfo.baseBranch)
```

Comment:
This line is already in `Promise.all()`. Do we need to call 'Promise.resolve()'?

Reply:
I missunderstood that `Promise.all()` takes array of `Promise` Object, but it seems that it can take no promises. I'll fix it not to use resolve. Thank you!
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all